src/MeetingsAdapter.test.js

```diff
@@ -0,0 +1,49 @@
+import {isObservable} from 'rxjs';
+
+import MeetingsAdapter from './MeetingsAdapter';
+
+describe('Meetings Adapter Interface', () => {
+  let meetingsAdapter;
+
+  beforeEach(() => {
+    meetingsAdapter = new MeetingsAdapter();
+  });
+
+  test('getMeeting() returns an observable', () => {
+    expect(isObservable(meetingsAdapter.getMeeting())).toBeTruthy();
+  });
+
+  test('getMeeting() throws a proper error message', (done) => {
```

Comment:
```suggestion
  test('getMeeting() errors because it needs to be defined', (done) => {
```
should you rephrase like the other tests?